### DISCUSSION TITLE
State of sleeping nodes on startup

### DISCUSSION THREAD

**Discussion by Vollervo** [zwave-js/zwave-js-ui#2403] [Q&A]
Hi,

Does anyone know how is the state of sleeping nodes determined on startup? I have two battery powered door sensors (of different brands) and upon restart one shows up as open while the other is closed. Is there a way to have the state as the last known state?

I'm running v6.5.2 inside Docker, BTW.

Thanks,

Vollervo

**Comment by robertsLando** [comment]
cc @AlCalzone

**Reply by AlCalzone** [reply]
That should be the last known state, unless the cache isn't persisted correctly.
